data-migrations/migrate-mongo-config.js

```diff
@@ -0,0 +1,36 @@
+// In this file you can configure migrate-mongo
+
+const config = {
+  mongodb: {
+    // TODO Change (or review) the url to your MongoDB:
+    url: "mongodb://localhost/test",
+
+    // TODO Change this to your database name:
+    databaseName: "test",
```

Comment:
same as https://github.com/courtcanva/cc-server/pull/8:

- remove useless comments
- URL and database name should be both environment variables